db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/SYSCOLUMNSRowFactory.java

```diff
@@ -507,6 +515,9 @@ public	int	getPrimaryKeyIndexNumber() {
                   null,null,view,viewId,0,0,0),
                 new ColumnDescriptor("SCHEMANAME"               ,15,15,
                         DataTypeDescriptor.getBuiltInDataTypeDescriptor(Types.VARCHAR, false, 128),
+                        null,null,view,viewId,0,0,0),
+                new ColumnDescriptor("SKETCHSIZE"               ,8,8,
```

Comment:
I think we should have `16,16` here because column 8 is already `AUTOINCREMENTVALUE`.